From 951caf42322fd2c40494379555e0d0d4faf102bb Mon Sep 17 00:00:00 2001
From: Gatien Chevallier <gatien.chevallier@foss.st.com>
Date: Tue, 2 Jan 2024 15:44:39 +0100
Subject: [PATCH 0303/1695] dts: stm32: enable risaf2/5 nodes for
 stm32mp257f-dk

Enables risaf2 and risaf5 nodes for this board.
These RISAF instances restrict accesses to OSPI and PCIE respectively.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 86933463acb86829a3702d9298b9193ed6a9c8c5 dts: stm32: enable risaf2/5 nodes for stm32mp257f-dk]
Signed-off-by: Gatien Chevallier <gatien.chevallier@foss.st.com>
Change-Id: Ib3b83a35cc703958751fd7b358ee4b4775a9dbb6
---
 core/arch/arm/dts/stm32mp257f-dk.dts | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/core/arch/arm/dts/stm32mp257f-dk.dts b/core/arch/arm/dts/stm32mp257f-dk.dts
index 22ddb2836..88cbf5a99 100644
--- a/core/arch/arm/dts/stm32mp257f-dk.dts
+++ b/core/arch/arm/dts/stm32mp257f-dk.dts
@@ -29,6 +29,14 @@
 	};
 };
 
+&risaf2 {
+	status = "okay";
+};
+
+&risaf5 {
+	status = "okay";
+};
+
 &usart2 {
 	pinctrl-names = "default";
 	pinctrl-0 = <&usart2_pins_a>;
-- 
2.47.2

